grid2op/Reward/_alertTrustScore.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) 2023, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
+
+import numpy as np
+from grid2op.Reward import AlertReward
+from grid2op.dtypes import dt_float
+
+SURVIVOR_TIMESTEPS = 12 #nb timesteps to be considered survivor of an attack
+
+class _AlertTrustScore(AlertReward):
+    """
+
+    INTERNAL
+
+    .. warning:: /!\\\\ Internal, do not use unless you know what you are doing /!\\\\
+            It **must not** serve as a reward. This scored needs to be **MAXIMIZED**,
+            as it is a negative! Also, this "reward" is not scaled or anything. Use it as your
+            own risk.
+
+    Implemented as a reward to make it easier to use in the context of the L2RPN competitions, this reward is based on the "alert feature" 
+    where the agent is asked to send information about potential line overload issue on the grid after unpredictable powerline
+    disconnection (attack of the opponent).
+    The alerts are assessed once per attack and cumulated over the episode. In this scheme, this "reward" computed the assistant "score",
+    which assesses how well the agent is aware of its capacities to deal with a situation during an episode.
+    It should not be used to train an agent.
+    
+    """
+
+    #TODO
+    #Parameters to use for challenge
+    #def __init__(self,
+    #             logger=None,
+    #             reward_min_no_blackout=-1.0,
+    #             reward_min_blackout=-50,
+    #             reward_max_no_blackout=0.0,
+    #             reward_max_blackout=0.0,
+    #             reward_end_episode_bonus=0.0,
+    #             min_score=-3):
+    def __init__(self,
+                 logger=None,
+                 reward_min_no_blackout=-1.0,
+                 reward_min_blackout=-10.0,
+                 reward_max_no_blackout=1.0,
+                 reward_max_blackout=2.0,
+                 reward_end_episode_bonus=1.0,
```

Comment:
can we put by default the parameters that are used for the competition ? Thanks

Reply:
ok